source/draw/gpu/opengl/backend/gles3/Gles3Fence.ooc

```diff
@@ -29,11 +29,11 @@ Gles3Fence: class extends GLFence {
 		version(debugGL) { validateEnd("Fence free") }
 		super()
 	}
-	clientWait: func (timeout: UInt64 = ULLONG_MAX) {
+	clientWait: func (timeout: UInt64 = ULLONG_MAX) -> Bool {
```

Comment:
Not strictly necessary, so you choose, but you used `override` for the other functions.